src/cargo/core/package_id.rs

```diff
@@ -160,14 +160,6 @@ impl PackageId {
         self.inner.source_id
     }
 
-    pub fn with_precise(self, precise: Option<String>) -> PackageId {
```

Comment:
Was this change meant to be in the next commit?

Reply:
It is dead code. Would you like me to split it out into a separate commit?